pkg/user/controllers/user.go

```diff
@@ -83,23 +90,28 @@ func (uc *UserController) Create(w http.ResponseWriter, r *http.Request) {
 // @Router /tokens [post]
 // CreateToken handles the server request and calls CreateTokenInteractor
 func (uc *UserController) CreateToken(w http.ResponseWriter, r *http.Request) {
-	var user entities.User
-	err := json.NewDecoder(r.Body).Decode(&user)
+	var param CreateTokenParam
+	err := json.NewDecoder(r.Body).Decode(&param)
 	if err != nil {
 		uc.logger.Error("failed to parse request body")
 		uc.writeResponse(w, http.StatusUnprocessableEntity, nil)
 		return
 	}
 
-	token, err := uc.createTokenInteractor.Execute(user)
+	tokenType := param.Type
+	if tokenType == "" {
+		tokenType = "user"
+	}
+
+	token, err := uc.createTokenInteractor.Execute(entities.User{Email: param.Email, Password: param.Password}, tokenType)
```

Comment:
Following @netoax's suggestion :point_down: 

```suggestion
	token, err := uc.createTokenInteractor.Execute(entities.User{Email: req.Email, Password: req.Password}, tokenType)
```

Reply:
Done!